### PR TITLE
Adds posters (for mapping)

### PR DESCRIPTION
**Pull request by AJCM-git**
## About the PR
Adds poster entities and spawners for mapping, functionality of rolled posters will be done in a future PR.

**Screenshots**
screenshot

### CODE REVIEW

**Comment by TimrodDX** [comment]
If you wouldn't mind, there's the "SPACE" poster that gets put near external airlocks which I don't think is in the game. Could you add that? I know there's at least a few of those on the original Packed and I'd be happy to put them on SS14 Packed.

**Comment by AJCM-git** [comment]
If you can find it and post the link sure

**Comment by AJCM-git** [comment]
Tests pass now and ready for a merge (couldn't find the poster Timrod was referring to). Thanks to @wrexbe for a solution to the MapInit issues

File: Resources/Prototypes/Entities/Objects/Decoration/posters.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- type: entity
+  parent: BaseItem
+  id: RolledPosterLegit
+  name: rolled legit poster
+  description: "An official Nanotrasen-issued poster to foster a compliant and obedient workforce. It comes with state-of-the-art adhesive backing, for easy pinning to any vertical surface."
+  components:
+  - type: Sprite
+    sprite: Structures/Wallmounts/posters.rsi
+    state: rolled_legit
```
Comment (PaulRitter) [inline Resources/Prototypes/Entities/Objects/Decoration/posters.yml:19]: afaict these are just placebo posters, which you cant actually place on walls. we decided on not adding non-functional items into the game, so these would either need to be removed or coded to actually be placeable

**Review by PaulRitter — CHANGES_REQUESTED** [review]
lgtm besides my comment

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml [hunk 2]
```diff
@@ -0,0 +1,823 @@
...
+    state: poster33_contraband
+
+- type: entity
+  parent: PosterBase
+  id: PosterContrabandLizard
+  name: "Lizard"
+  description: "This lewd poster depicts a lizard preparing to mate."
+  components:
+  - type: Sprite
+    state: poster34_contraband
```
Comment (PaulRitter) [inline Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml:354]: remove this one & its image

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml [hunk 3]
```diff
@@ -0,0 +1,823 @@
...
+    state: poster33_legit
+
+- type: entity
+  parent: PosterBase
+  id: PosterLegitNoERP
+  name: "No ERP"
+  description: "This poster reminds the crew that Eroticism, Rape and Pornography are banned on Nanotrasen stations."
+  components:
+  - type: Sprite
+    state: poster34_legit
```
Comment (PaulRitter) [inline Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml:796]: remove this one and its image

**Review by PaulRitter — CHANGES_REQUESTED** [review]
otherwise lgtm

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml (hunk 3 above)
Comment (PaulRitter) [reply Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml:796]: scratch that, just tone down the description

**Comment by AJCM-git** [comment]
Reviews addressed

**Review by PaulRitter — APPROVED** [review]

**Pull request merged**